mock/mockSyfotekster.js

```diff
@@ -15,15 +16,19 @@ function hentTeksterFraProd() {
         }
     });
 }
+*/
 
 function mockTekster(server) {
+/*
     const HVERT_FEMTE_MINUTT = 1000 * 60 * 5;
     hentTeksterFraProd();
     setInterval(hentTeksterFraProd, HVERT_FEMTE_MINUTT);
+*/
 
     server.get('/syfotekster/api/tekster', (req, res) => {
         res.setHeader('Content-Type', 'application/json');
-        res.send(JSON.stringify(teksterFraProd || mockData[enums.TEKSTER]));
+        // res.send(JSON.stringify(teksterFraProd || mockData[enums.TEKSTER]));
```

Comment:
Er dette med vilje?

Reply:
Ja, siden det fremdeles er litt usikkerhet rundt håndtering av tekster, så tenkte jeg det var greit å ha en enkel vei tilbake - inntil videre.

Reply:
skjønner.. Husk å synce med Veden at dette evt. skjer. De har jo også et forhold til disse tekstene

Reply:
Jeg er mer fan av å fjerne koden i stedet for å kommentere den ut. Da slipper man å forholde seg til kommentert kode på ubestemt tid og koden ligger i git-historikken.

Reply:
Enig med Anders. Du kan fjerne linjene i egen commit så kan bare den rulles tilbake hvis vi vil tilbake til den gamle måten.